pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -261,6 +261,12 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 				}
 			}
 		}
+		// externalTrafficPolicy: Local is unsupported by OVN in
+		// OpenShift 4.9 and earlier; see
+		// <https://bugzilla.redhat.com/show_bug.cgi?id=2060542>.
+		if networkType == string(operatorv1.NetworkTypeOVNKubernetes) {
+			service.Spec.ExternalTrafficPolicy = "Cluster"
```

Comment:
nit: we could use the https://github.com/kubernetes/kubernetes/blob/e6c093d87ea4cbb530a7b2ae91e54c0842d8308a/pkg/apis/core/types.go#L3672 constant from upstream

Reply:
Fixed.  Thanks!